### PR TITLE
che#1794 add simple language server for test purpose

### PR DESCRIPTION
**Pull request by evidolob**
Signed-off-by: Yevhen Vydolob <evidolob@codenvy.com>

### What does this PR do?
add simple language server for test purpose

### What issues does this PR fix or reference?
che#1794

### CODE REVIEW

File: che-test-ls/src/server.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Codenvy, S.A. - initial API and implementation
+#
+
+cd "${0%/*}"
+node ./server.js --stdio
```
Comment (vparfonov) [inline che-test-ls/src/server.sh]: new line

File: .gitignore [hunk 2]
```diff
@@ -55,4 +55,6 @@ ehthumbs.db
 Icon?
 Thumbs.db
 */overlays
-*~
\ No newline at end of file
+*~
+
+/**/node_modules/
```
Comment (vparfonov) [inline .gitignore]: new line

**Comment by vparfonov** [comment]
Don't merge without @skabashnyuk approve

**Comment by skabashnyuk** [comment]
CQ for build and test deps please

**Comment by evidolob** [comment]
CQ https://dev.eclipse.org/ipzilla/show_bug.cgi?id=13958

**Pull request merged**